pkg/gui/controllers/helpers/app_status_helper.go

```diff
@@ -33,13 +33,29 @@ func (self *AppStatusHelper) Toast(message string) {
 	self.renderAppStatus()
 }
 
+// A custom task for WithWaitingStatus calls; it wraps the original one and
+// hides the status whenever the task is paused, and shows it again when
+// continued.
+type appStatusHelperTask struct {
```

Comment:
Can we make the implementation of gocui.Task explicit via:
`var gocui.Task _ = appStatusHelperTask{}` above? (I know it's almost explicit via embedding the interface but that's not perfect because overridden methods can have different signatures)

Reply:
Not exactly sure if you meant this: eb20f3c0ec94